lib/open_food_network/reports/variant_overrides.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module OpenFoodNetwork
+  module Reports
+    class VariantOverrides
```

Comment:
I think there's a better and more reusable structure for this code.
I think this should be a service class in app/services (it's not related to reports in its implementation only in its usage).
According to our convention the resource name of the service should be first, so something like VariantOverridesIndex or VariantOverridesIndexed.
The cool thing about this move is that you can move VariantOverride#indexed (the original one) into this service as well (this can be done in a later move/PR).

Reply:
Done